### DISCUSSION TITLE
Yunikorn and volcano scheduling

### DISCUSSION THREAD

**Discussion by paulpaul1076** [stackabletech/spark-k8s-operator#311] [Q&A] [answered]
Is it possible to integrate stackable trino and spark operators with yunikorn or volcano schedulers?

**Comment by adwk67** [comment]
Thanks for your question! We are planning this for at least spark (see [this](https://github.com/stackabletech/spark-k8s-operator/issues/158) issue), but this is more a mid-/long-term priority (at the moment).

**Comment by adwk67** [comment, marked as answer]
There are also some notes on made on Yunikorn and related things [here](https://github.com/stackabletech/spark-k8s-operator/issues/121). It my be possible to use Yunikorn without any changes to our resource definitions.
